pkg/manager/daemon_adaptor.go

```diff
@@ -160,6 +165,17 @@ func (m *Manager) BuildDaemonCommand(d *daemon.Daemon, bin string, upgrade bool)
 			command.WithID(d.ID()))
 	}
 
+	prefetchMap := prefetch.GetPrefetchMap()
+	if imageReference != "" {
+		prefetchfiles, ok := prefetchMap[imageReference]
```

Comment:
Accessing prefetchMap should be locked.

Reply:
fixed， I have add a lock in `GetPrefetchMap` and copy value in it.